src/plugins/main.test.ts

```diff
@@ -134,6 +135,68 @@ describe('default values', () => {
   });
 });
 
+describe('isInvalid method', () => {
+  // Silence expected error logging and restore when finished.
+  const stderr = console.error;
+  beforeAll(() => (console.error = () => {}));
+  afterAll(() => (console.error = stderr));
+
+  test('basic plugin', () => {
+    const game: Game = {
+      plugins: [
+        {
+          name: 'test',
+          isInvalid: ({ G }) => 'bad' in G && 'not ok',
+        },
+      ],
+      moves: {
+        good: () => ({ good: 'nice' }),
```

Comment:
NIT move the {good: 'nice'} to a constant so we you dont need to repeat it when checking later?